x/wasm/keeper/gas_register.go

```diff
@@ -52,6 +52,8 @@ const (
 	DefaultPerCustomEventCost uint64 = 20
 	// DefaultEventAttributeDataFreeTier number of bytes of total attribute data we do not charge.
 	DefaultEventAttributeDataFreeTier = 100
+	// DefaultPerByteUncompressCost is how much SDK gas we charge per source byte to unpack
+	DefaultPerByteUncompressCost uint64 = 2
```

Comment:
You can make this of type `wasmvmtypes.UFraction` and change to 15/100 (=0.15).

1 gas or more per byte is very expensive for gzip decompression.